terraform/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "oidc-gatekeeper"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
+}
+
+variable "config" {
+  description = "Map of charm configuration options"
+  type        = map(string)
+  default     = {}
+}
+
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+# TODO: Update to a map of strings, once juju provider 0.14 is released
+variable "resources" {
+  description = "Map of resources revisions"
+  type        = map(number)
+  default     = null
+}
```

Comment:
Need to be updated https://github.com/canonical/minio-operator/pull/188/files#r1764625697